test/molecule-role/molecule/default/prepare.yml

```diff
@@ -108,6 +117,12 @@
         name: nc
         state: present
       when: ansible_distribution == "Fedora"
+    - name: Install libselinux-python in Fedora
```

Comment:
why do we need libselinux on the agent ?

Reply:
I was getting this error "Aborting, target uses SELinux but python bindings (libselinux-python) aren’t installed" and found this blog `https://medium.com/@audun.nes/aborting-target-uses-selinux-but-python-bindings-libselinux-python-arent-installed-f6b9d4542130` which says it comes during molecule test.